src/main/resources/i18n/Validation.properties

```diff
@@ -98,3 +100,6 @@ ArgumentValidationUtil.handleNotObjectError=Validation error ({0}) : argument ''
 ArgumentValidationUtil.handleMissingFieldsError=Validation error ({0}) : argument ''{1}'' with value ''{2}'' is missing required fields ''{3}''
 # suppress inspection "UnusedProperty"
 ArgumentValidationUtil.handleExtraFieldError=Validation error ({0}) : argument ''{1}'' with value ''{2}'' contains a field not in ''{3}'': ''{4}''
+
+DirectiveMisplaced.notAllowedOperationRootLevel=Validation error ({0}) : Directive ''{1}'' is not allowed on root of operation ''{2}''
```

Comment:
Since these messages are specific to `@defer` (and in the future maybe shared with `@stream`), should we use a more explicit prefix for these i18n keys?
Maybe `IncrementalDirectiveMisplaced`?

Reply:
I agree these should have a specific prefix - I cant see these messages being re-used